allennlp/data/tokenizers/pretrained_transformer_tokenizer.py

```diff
@@ -63,16 +64,19 @@ def __init__(
         stride: int = 0,
         truncation_strategy: str = "longest_first",
         calculate_character_offsets: bool = False,
+        tokenizer_kwargs: Dict[str, Any] = {},
     ) -> None:
-        self.tokenizer = AutoTokenizer.from_pretrained(model_name)
+        self.tokenizer = AutoTokenizer.from_pretrained(model_name, **tokenizer_kwargs)
 
         # Huggingface tokenizers have different ways of remembering whether they lowercase or not. Detecting it
         # this way seems like the least brittle way to do it.
         tokenized = self.tokenizer.tokenize(
             "FOO"
         )  # Use a short word that's unlikely to be cut into word pieces.
         detokenized = " ".join(tokenized)
-        self._tokenizer_lowercases = "foo" in detokenized
+        self._tokenizer_lowercases = (
```

Comment:
I don't believe this change is giving us anything extra, is it?

Reply:
I believe it actually changes heuristic approach, which can fail to a better one.